coverage-tests.js

```diff
@@ -2162,6 +2162,30 @@ test('should return aborted tests in getAllTestResults', {
   },
 })
 
+test('Should not fail all by some aborted tests', {
```

Comment:
I think we should keep consistency with the naming. We don't have names that start with the capital latter.

Reply:
np, it's a 'copy-paste' typo... fixed the original (the one I copied from) and mine